index.d.ts

```diff
@@ -44,6 +44,11 @@ declare namespace activeWin {
 		*/
 		owner: BaseOwner;
 
+		/**
+		URL of active browser tab if the active window is Safari, Chrome, or Brave.
+		*/
+		url: string;
```

Comment:
```suggestion
		url?: string;
```

Reply:
And it should be moved to https://github.com/robertjkeck2/active-win/blob/e2edaea2ad79636687a01b6ce65f4d7d700d907b/index.d.ts#L65-L69 as it's only available on macOS.

Reply:
Updated and moved to `MacOSResult` interface.